README.md

```diff
@@ -199,8 +199,16 @@ Similarly to OpenAI Gym, Jumanji keeps a strict versioning of its environments f
 We maintain a registry of standard environments with their configuration.
 For each environment, a version suffix is appended, e.g. `Snake-6x6-v0`.
 When changes are made to environments that might impact learning results,
-the version number is incremented by one to prevent potential confusion. For a full list of registered versions
-of each environment, check out the environments' documentation.
+the version number is incremented by one to prevent potential confusion. Jumanji currently provides the following registered versions for each environment:
+
+- 🐍 Snake: `Snake-6x6-v0`, `Snake-12x12-v0`.
+- 4️⃣ Connect4: `Connect4-v0`.
+- 📬 TSP: `TSP50-v0`, `TSP100-v0`, `TSP150-v0`, `TSP200-v0`.
+- 🎒 Knapsack: `Knapsack50-v0`, `Knapsack100-v0`, `Knapsack200-v0`, `Knapsack250-v0`.
+- 🪢 Routing: `Routing-n3-8x8-v0`, `Routing-n4-12x12-v0`, `Routing-n5-16x16-v0`.
+- 📦 BinPack: `BinPack-toy-v0`, `BinPack-rand20-v0`, `BinPack-rand40-v0`, `BinPack-rand100-v0`.
+
+For more details about each registered environment, [see the documentation](https://instadeepai.github.io/jumanji/environments/tsp/).
```

Comment:
Do we want this pointing to TSP?

Reply:
By default, when we click on the "Environments" section, it starts at TSP, `https://instadeepai.github.io/jumanji/environments` will give a 404. @clement-bonnet and I discussed and we could add another markdown file so that it doesn't default to TSP (which happens to be the first environment under the "Environments" section).